examples/31-serving-custom-store.yaml

```diff
@@ -1,42 +1,44 @@
 apiVersion: numaflow.numaproj.io/v1alpha1
-kind: Pipeline
+kind: ServingPipeline
 metadata:
   name: serving-pipeline
 spec:
-  servingStore:
-    name: redis
-    container:
-      image: quay.io/numaio/numaflow-go/serving-redis-store:stable
+  serving:
+    service: true
+    msgIDHeaderKey: "X-Numaflow-Id"
+    store:
+      container:
+        image: quay.io/numaio/numaflow-go/serving-redis-store:stable
+  pipeline:
+    vertices:
+      - name: serving-in
+        scale:
+          min: 1
+        source:
+          serving:
+            service: true
```

Comment:
The examples need to be cleaned up later on.

Reply:
@yhl25 you want to tackle this in this PR?